spec/ParseAPI.spec.js

```diff
@@ -382,29 +382,6 @@ describe('miscellaneous', function() {
     });
   });
 
-  it('test rest_create_app', function(done) {
```

Comment:
This test is extremely important, since it's required for client integration tests to pass.
Please revert.